clang/test/3C/canwrite_constraints.h

```diff
@@ -19,3 +19,9 @@ int *foo_var = ((void *)0);
 // dedicated test for it.
 inline void no_op() {}
 // CHECK_HIGHER: inline void no_op() _Checked {}
+
+// In the lower case, this should stay wild
+// In the higher case, this should solve to checked
+typedef int* intptr;
```

Comment:
Is there something you could add to `base_subdir/canwrite_constraints.c` that would be rewritten differently depending on whether 3C is able to modify this typedef or not (analogous to how `foo` and `foo_var` are used in `bar` now)?  If not, can you please add a comment there explaining why not?